src/main/resources/SQLQuery1.sql

```diff
@@ -0,0 +1,91 @@
+CREATE database iCareNow;
+
+USE iCareNow;
+
+DROP TABLE IF EXISTS icn_role
```

Comment:
As far as i remember i asked alina to create a new branch and pr on this changes. Please remove and ask her to add it separately